Tiltfile

```diff
@@ -1,9 +1,3 @@
 print('dissolution-web')
 
 include(path = './Tiltfile.dev')
-
-docker_compose(
```

Comment:
why remove this?

Reply:
The docker-compose file in dissolution-web isn't used (the one in docker-chs-development is used), so that's why in dissolution-web we can delete the docker-compose file and delete the invoking of it in the Tilefile

Reply:
Hmmm ok - would be good to compare with lfp-appeals-frontend and suppression-web. Don't want our auto recompile to suddenly stop working in tilt

Reply:
dissolution-web still works ok and auto recompiles without it. But yeah lfp-appeals-frontend and suppression-web still has separate docker-compose files in their repos.

@geek-soft Do you know if https://github.com/companieshouse/dissolution-web/blob/master/docker-compose.yaml has any purpose? I assumed Tilt is only using https://github.com/companieshouse/docker-chs-development/blob/master/services/modules/dissolution/dissolution-web.docker-compose.yaml to spin up the dissolution-web image?

Reply:
@geek-soft ^